robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -158,6 +158,29 @@ def pause_recurring_donation(self, **kwargs):
         self.selenium.click_element(locator)
         self.salesforce.wait_until_modal_is_open()
 
+    @capture_screenshot_on_error
+    def populate_pause_modal(self,**kwargs):
+      for key, value in kwargs.items():
```

Comment:
Please add documentation to this keyword

Reply:
please run robot_libdoc task

Reply:
Done